packages/relay-compiler/bin/RelayCompilerMain.js

```diff
@@ -23,6 +23,7 @@ const WatchmanClient = require('../core/GraphQLWatchmanClient');
 
 const crypto = require('crypto');
 const fs = require('fs');
+const glob = require('glob');
```

Comment:
Is there a reason to always require this rather than inline as before?

Reply:
I assume it doesn't matter but I don't know.

Reply:
I guess it was inlined because this code-path isn’t hit when you use watchman?

Reply:
Not sure if we do it for open-source builds, but internally at FB we automatically inline `require`s so the behaviour would be identical in that case.

Reply:
iirc inlining all requires was disabled, as it led to worse behaviour of relay-runtime in a SSR context. /cc @robrichard